templates/docs/patterns/tiered-list/index.md

```diff
@@ -19,13 +19,13 @@ variety of tiered list layouts:
 
 The tiered list pattern is composed of the following elements:
 
-| Element               | Description                                                       |
-| --------------------- | ----------------------------------------------------------------- |
-| Title (**required**)  | <code>h2</code> title text                                        |
-| Description           | <code>p</code> description text                                   |
-| List item title       | Title text/content                                                |
-| List item description | Description text/content                                          |
-| Call to action block  | [Call to action block](/docs/patterns/cta-block) beneath the list |
+| Element               | Description                                                            |
+| --------------------- | ---------------------------------------------------------------------- |
+| Title (**required**)  | <code>h2</code> title text                                             |
+| Description           | <code>p</code> description text; [CTA block](/docs/patterns/cta-block) |
+| List item title       | Title text/content                                                     |
+| List item description | Description text/content; [CTA block](/docs/patterns/cta-block)        |
```

Comment:
Reading these on the docs page, it's a bit unclear that the CTA block is being suggested as optional content to pass into a description. Maybe this can be edited to say something like "with optional CTA block" or similar?

<img width="750" alt="Screenshot 2024-08-21 at 3 21 07 PM" src="https://github.com/user-attachments/assets/46018ec1-3a58-41aa-bb1b-c7aea4c474fa">